### PR TITLE
Fix formatting in getEditorSettings docs and update getTokenSettings docs.

### LINKED ISSUE #10593: Documentation: 2 JSDoc blocks missing double asterisk
See

https://github.com/WordPress/gutenberg/blob/1478d14292afc4d6edef8e661673cc083c8c3d43/packages/editor/src/store/selectors.js#L1931-L1955

Actually these two blocks are not parsed as documentation blocks. For example, ESLint doesn't lint them. As soon as the missing asterisk is added, ESLint will warn about missing documentation for the `name` parameter in `getTokenSettings()`.

Minor: the `@return` line should end with a period.

Adding the missing asterisk is trivial and I could fix it myself, but I'm not sure the `getTokenSettings()` documentation is 100% correct. I guess it should be changed as it relates to the available inline Blocks and not to the editor settings.  /Cc @aduth

### PR DESCRIPTION
**Pull request by brentswisher**
## Description
Added formatting change to include getEditorSettings and getTokenSettings in lint-js, update getTokenSettings docs to not be a copy of getEditorSettings and add name param. Fixes #10593. I gave my best shot at the getTokenSettings docs, but I am still exploring the gutenberg code so if @aduth or anyone else sees something I missed or got wrong, feel free to update.  

## How has this been tested?
Fixiing the missing second asterisk added getEditorSettings and getTokenSettings to the lint-js task, adding the name param fixed the error it reported.

## Screenshots
N/A

## Types of changes
Documentation fix and update

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation.

### CODE REVIEW

File: packages/editor/src/store/selectors.js [hunk 1]
```diff
@@ -1928,23 +1928,24 @@ export function getBlockListSettings( state, clientId ) {
...
-/*
- * Returns the editor settings.
+/**
+ * Returns the token settings.
  *
  * @param {Object} state Editor state.
+ * @param {?string} name Token name.
  *
- * @return {Object} The editor settings object
+ * @return {Object} Token settings object, of the named token if set.
```
Comment (aduth) [inline packages/editor/src/store/selectors.js]: Typo: "of" -> "or"

**Review by aduth — CHANGES_REQUESTED** [review]
You need to run `npm run docs:build` and commit the updated files for the build to pass. Otherwise this is looking nice!

**Comment by brentswisher** [comment]
Thanks for the feedback, I generated the markdown file and fixed the typo.

**Review by aduth — APPROVED** [review]
Looks great, thanks 👍

**Pull request merged**